scripts/console.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+docker-compose run web ./manage.py shell
```

Comment:
These are great; PFB has a pretty complete example of using the STRTA pattern within a project; no need to make changes to these, but that's the type of thing we would normally build for a client project.